### DISCUSSION TITLE
FYI: Easy installation on Windows 11

### DISCUSSION THREAD

**Discussion by Xeevis** [ndbeals/winssh-pageant#29] [Show and tell]
In README it currently calls to install builds of OpenSSH that are mostly for non-production and also to add OpenSSH to System PATH.

>Prerequisites
Install the [Microsoft OpenSSH package, found on their Github](https://github.com/PowerShell/Win32-OpenSSH/releases). Do not install this using Windows update, that one is quite outdated, and will not work with this software. [Follow the instructions for adding OpenSSH to your System PATH.](https://github.com/PowerShell/Win32-OpenSSH/wiki/Install-Win32-OpenSSH-Using-MSI)

Not sure how far that's still relevant, but on latest Windows 11 and PowerShell, all I had to do:
```powershell
Get-Service ssh-agent | Start-Service -PassThru | Set-Service -StartupType Automatic
winget install NathanBeals.WinSSH-Pageant -e
```
WinSCP starts working with keys added by KeePassXC OpenSSH agent integration. No hiccups whatsoever.

**Comment by ndbeals** [comment]
That's excellent news! Before I edit the README would you mind running the following command in powershell and pasting the results of it back here?

`Get-Command ssh`

Thanks.

**Reply by Xeevis** [reply]
```
CommandType    Name      Version    Source
-----------    ----      -------    ------
Application    ssh.exe   8.6.0.1    C:\Windows\System32\OpenSSH\ssh.exe
```

**Reply by Xeevis** [reply]
I see it's not the newest version, but appears to be working fine, is there known incompatibility? Anything that I can test?

**Reply by ndbeals** [reply]
You're right, it's not the newest version, but it is a version new enough to not have any (known) issues. 

There's nothing you need to test, I just wanted to ensure microsoft finally did update the openssh package you get through windows update, seems they did.

I'll update the README soon. Thanks for opening this ticket.
